### ISSUE TITLE
[MAS2.4.4] [Screen reader -Endpoint] “Learn more ” link is not descriptive 

### ISSUE THREAD

**Issue by Kumar2608** [microsoft/BotFramework-Emulator#1795]
**Actual Result:**
“Learn more” link is not descriptive present in the Add an endpoint for your bot dialog 
Learn more link is reading as Learn more link 

**Expected Result:**
Learn more link should be more descriptive like Learn more about Azure for us government
For every link should more descriptive .

**Note:** Same issue repro for all Learn more links 

**Repro Steps:**
1.Open the Bot Framework V4 Emulator.
2.Navigate to the Bot explorer and select it 
3.Navigate to the End point symbol control and select it 
4.Dialog gets opened 
5.Navigate to the Learn more link 
6.Observe the issue 

**User Impact:**
Users who rely on Screen Readers will be affected to navigate the links if Narrator is not narrating the descriptive info for link.

**Test Environment:**
OS Build: Version 1903(OS Build 18362.239)
Application: Bot Framework V4 Emulator
Screen reader: NVDA 

**MAS Reference Link:**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={06a891af-210c-4c6d-bb3e-92babab209e7}
[MAS2.4.4_End point.zip](https://github.com/microsoft/BotFramework-Emulator/files/3559356/MAS2.4.4_End.point.zip)
